src/main/resources/init_db.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE `books` (
+`id` BIGINT NOT NULL AUTO_INCREMENT,
```

Comment:
Actually MySQL declares primary key as NOT NULL automatically
PRIMARY KEY
A unique index where all key columns must be defined as NOT NULL. If they are not explicitly declared as NOT NULL, MySQL declares them so implicitly (and silently).